examples/spark/spark-gluster/glusterfs-endpoints.yaml

```diff
@@ -0,0 +1,14 @@
+kind: Endpoints
+apiVersion: v1
+metadata: 
+  name: glusterfs-cluster
+subsets: 
+  - addresses: 
+      - IP: 192.168.58.218
```

Comment:
IP should be lowercase to match the struct tag here https://github.com/kubernetes/kubernetes/blob/master/pkg/api/v1/types.go#L1671